benchmarks/src/gatling/scala/org/apache/polaris/benchmarks/actions/TableActions.scala

```diff
@@ -184,6 +184,7 @@ case class TableActions(
     http("Fetch Table")
       .get("/api/catalog/v1/#{catalogName}/namespaces/#{multipartNamespace}/tables/#{tableName}")
       .header("Authorization", "Bearer #{accessToken}")
+      .header("If-None-Match", "")
```

Comment:
is the ETAG purely random or its in the format Polaris generates ? 
looks very similar to https://github.com/apache/polaris/blob/main/service/common/src/main/java/org/apache/polaris/service/http/IcebergHttpUtil.java#L45

Reply:
This is the same format that Polaris generates pretty much. From the etag example in the description, I think it's from Polaris. Unfortunately setting it as `""` did not clear it.

Reply:
https://groups.google.com/g/gatling/c/oYlzjJTAg1U

Reply:
Yeah I think that's exactly right @dimas-b